### PR TITLE
 Fix hyperlinks to repositories

### PR DESCRIPTION
**Pull request by amulware**
Adds a link to the toggl mobile repo, and fixes the one to superday (which was broken since we added the bitrise badge).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,6 +16,6 @@ This repository contains documentation relevant for anyone in the Toggl mobile t
 
 Each app may have additional app-specific documentation, like architectural and code style guidelines. You can find those linked from the respective apps' readme files:
 
-- Toggl Mobile
-- [Supertoday](https://github.com/toggl/superday#-superday "Superday repository")
-- [Stopwatch](https://github.com/toggl/stopwatch/blob/master/Readme.md#toggl-ios-stopwatch-app "Toggl iOS Stopwatch App")
\ No newline at end of file
+- [Toggl Mobile](https://github.com/toggl/mobileapp#toggl-mobile-app- "Toggl Mobile App repository")
+- [Supertoday](https://github.com/toggl/superday#-superday- "Superday repository")
```
Comment (Odrakir) [inline README.md]: Can't we call it "Superday" already?

**Comment by amulware** [comment]
Waiting with this until the name change is confirmed.

**Comment by amulware** [comment]
Do you mind, @Odrakir?

**Comment by Odrakir** [comment]
Not at all 😊

**Pull request merged**